BUILD.bazel

```diff
@@ -12,6 +12,7 @@ load("@bazel_gazelle//:def.bzl", "gazelle")
 #
 # gazelle:prefix github.com/cockroachdb/cockroach
 # gazelle:build_file_name BUILD.bazel
+# gazelle:build_tags bazel
```

Comment:
Doesn't count for much but LGTM. If anything, maybe we could use a more greppable build tag here should things get unwieldy later on.